website/blog/2018-04-30-How-I-Converted-Profilo-To-Docusaurus.md

```diff
@@ -0,0 +1,120 @@
+
+**Content creation:**
+
+1. Added metadata to the existing Markdown files found in the `docs` folder, for example:
+
+        +---
+        +id: architecture
+        +title: Architecture
+        +sidebar_label: Architecture
+        +---
+
+1. Added the logo assets to the `website/static/img` folder.
+1. Modified `website/pages/en/index.js`, the landing page, to highlight Profilo features.
+1. Modified `website/core/Footer.js`, the footer, to simplify it for Profilo.
+1. Edited `website/siteConfig.js` (website configuration file) to specify the previously chosen primary and secondary colors.
+1. Modified `website/sidebars.json` that specifies the sidebar navigation. Listed all the docs and customized it based on the metadata added to the Markdown files.
+1. Edited the website configuration file to specify the GitHub properties, logo images, header links, and the website link.
+1. Tested the website locally throughout this phase. (I ran `yarn start` from the `website` folder to start the server.)
+
+**Feedback and review changes:**
+
+1. Sent a [pull request](https://github.com/facebookincubator/profilo/pull/6) to the project.
+1. Updated the colors after the designer rightly gasped at the ones I had chosen (IANAD).
+1. Updated the colors and updated the PR.
+1. The PR was then accepted and [merged](https://github.com/facebookincubator/profilo/commit/6ad033aaf5a7d54e6d842f45a5bccd051a8e45ad). Yay!!
+
+**Website publishing:**
+
+1. Pushed the first website version by running the Docusaurus publish script from the command line:
+
+        USE_SSH=true \
+          GIT_USER=caabernathy \
+          CURRENT_BRANCH=master \
+          yarn run publish-gh-pages
+
+1. Configured Circle CI using the [provided Docusaurus instructions](https://docusaurus.io/docs/en/publishing.html#automating-deployments-using-continuous-integration). There were 2 PRs for this, [the first](https://github.com/facebookincubator/profilo/pull/8)for the initial config and [the second](https://github.com/facebookincubator/profilo/pull/12) to make sure Circle CI only triggered for changes in the master branch (thanks Joel Marcey!).
+
+The final website was published on https://facebookincubator.github.io/profilo/. It had taken 1.5 hours to get to the initial PR stage and another half an hour or so to respond to review feedback and publish the website.
+
+## Design
+
+Here's what the initial website looked like when the first pull request was sent out:
+
+![Website Initial Design](/img/profilo_blog_post_website_initial.png)
+
+Most of the time spent in the content creation was spent picking colors that worked reasonably well with the given logo. These colors were a good jumping off point for designer feedback. I used Photoshop came to sample various portions of the logo.
```

Comment:
> I used Photoshop came to sample various portions of the logo.

I don't understand this line.

Reply:
I think I fixed that with my latest commit.